internal/pkg/agent/vault/vault_windows.go

```diff
@@ -59,42 +65,90 @@ func New(path string, opts ...OptionFunc) (v *Vault, err error) {
 		}
 	}
 
-	entropy, err := getOrCreateSeed(path, options.readonly)
+	r := &Vault{
```

Comment:
[NIT]
Why are you using `r` instead of `v`?

Reply:
want to return v as nil if err happens after this line